src/lib/viewers/controls/color-picker/ColorPickerControl.tsx

```diff
@@ -1,16 +1,19 @@
 import React, { useState } from 'react';
 import classNames from 'classnames';
 import ColorPickerPalette from './ColorPickerPalette';
+import { AnnotationColor } from '../../../AnnotationModule';
 import { AnnotationMode } from '../annotations/types';
 import './ColorPickerControl.scss';
 
 export type Props = {
+    annotationColor: AnnotationColor;
     annotationMode?: AnnotationMode;
```

Comment:
Had left the comment on a previous PR, but does `annotationMode` need to live here? A color picker doesn't seem to need to know about annotations at all

Reply:
I can remove it from this component and perform the null check in DocControls instead.